site/content/blog/2021-10-01-whats-new-in-svelte-october-2021.md

```diff
@@ -0,0 +1,98 @@
+---
+title: What's new in Svelte: October 2021
+description: A whole year of "What's new in Svelte"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Hey y'all 👋 It's been 1 year since "What's new in Svelte" started being cross-posted to the Svelte blog. I wanted to take this moment at the top to thank all of you for reading and for all the contributors every month. From the maintainers to everyone who posts their work in the Discord and Reddit, it's amazing to witness all the effort that goes into making the Svelte community great.
+
+Keep up the good work, everyone! Now, let's dive into this month's news...
+
+## New around Svelte
+
+- New additions to Svelte's export map now expose no-op versions of lifecycle functions for SSR (Svelte **3.43.0**)
+- Custom components with a `src` attribute no longer break `svelte-native` builds (Svelte **3.42.4**)
+- Svelte plugin users without [the TypeScript plugin](https://www.npmjs.com/package/typescript-svelte-plugin) enabled will now be prompted to enable it. It enhances TypeScript and JavaScript files with additional intellisense to interact with Svelte files. [Please leave feedback](https://github.com/sveltejs/language-tools/issues/580) if you are using it (Svelte extensions **105.4.0**)
+- Event modifiers have been added to intellisense as autocomplete and hover info (Svelte extensions **105.4.0**)
+- TypeScript users no longer have to strictly separate type imports and value imports when using Svelte version 3.39 or higher and `svelte-preprocess` version 4.9.5 or higher. This means you can now write `import { MyInterface, myValue } from './somewhere` instead of `import type { MyInterface } from './somewhere'; import { myValue } from './somewhere`. Huge thanks to community member [@SomaticIT](https://github.com/SomaticIT) who mainly implemented this!
```

Comment:
```suggestion
- TypeScript users no longer have to strictly separate type imports and value imports when using Svelte version 3.39 or higher and `svelte-preprocess` version 4.9.5 or higher. This means you can now write `import { MyInterface, myValue } from './somewhere'` instead of `import type { MyInterface } from './somewhere'; import { myValue } from './somewhere'`. Huge thanks to community member [@SomaticIT](https://github.com/SomaticIT) who mainly implemented this!
```